magento/adobe-stock-integration
# 1618: Duplicated Items in Used in Entities filters

### Description (*)

This PR will introduce the fix to the issue wherein the options in Used In Entities filter are being duplicated.
### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration https://github.com/magento/adobe-stock-integration/issues/1618: Duplicated items in Used in Entities filters

### Manual testing scenarios (*)


1. Go to **Content - Media Gallery**
2. Click **Filters**
3. Expand, for example, **Used in Pages** filter
4. Start typing _page_ into **Page Title** field
5. The options displayed should not be duplicated.
